core/src/main/java/io/kestra/scheduler/events/TriggerCompleted.java

```diff
@@ -0,0 +1,21 @@
+package io.kestra.scheduler.events;
+
+import io.kestra.core.models.flows.State;
+import io.kestra.core.models.triggers.TriggerId;
+
+import java.time.Instant;
+
+/**
+ * A trigger execution completed.
+ */
+public record TriggerCompleted(
```

Comment:
```suggestion
public record TriggerExecutionTerminated(
```

Be explicit.
We don't use completed but terminated execution